administrator/language/de-DE/de-DE.plg_system_logrotation.ini

```diff
@@ -4,9 +4,9 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
 ; Note : All ini files need to be saved as UTF-8
 
-PLG_SYSTEM_LOGROTATION="System - Log Rotation"
-PLG_SYSTEM_LOGROTATION_XML_DESCRIPTION="This plugin periodically rotates system log files."
-PLG_SYSTEM_LOGROTATION_CACHETIMEOUT_LABEL="Log Rotation (in days)"
-PLG_SYSTEM_LOGROTATION_CACHETIMEOUT_DESC="How often should the logs be rotated."
-PLG_SYSTEM_LOGROTATION_LOGSTOKEEP_DESC="The maximum number of old logs to keep."
-PLG_SYSTEM_LOGROTATION_LOGSTOKEEP_LABEL="Maximum Logs"
+PLG_SYSTEM_LOGROTATION="System - Protokollrotation"
+PLG_SYSTEM_LOGROTATION_XML_DESCRIPTION="Dieses Plugin rotiert periodisch die Systemprotokolldateien."
+PLG_SYSTEM_LOGROTATION_CACHETIMEOUT_LABEL="Protokollrotation (in Tagen)"
```

Comment:
Protokollrotationsdauer? (Da wir einen Zeitraum in Tagen angeben)

Reply:
Finde das so klarer und verständlicher

Reply:
ok.